docs/logical-data-model.md

```diff
@@ -0,0 +1,80 @@
+# Logical data model
+
+![Resource](./assets/logical-data-model-1.png)
+
+## Resource
+
+| Key | Value description |
+| --- | ----------------- |
+| `id` | A unique resource id. |
+| `parentId` | The parent resource id. |
+| `version` | Resource version. |
+| `title` | The name (title) of the resource. Rendered in the UI. |
+| `description` | The description of the resource. Rendered in the UI. |
+| `resourceType` | Resource type. Workspace or Service. |
+| `status` | Resource status. |
+| `markedForDeletion` | Is the resource marked for deletion. |
```

Comment:
we currently have an isDeleted in the API - is this the same? i.e. is this for deleting the resource request or for deleting the underlying resources?

Reply:
I changed to isDeleted